app/models/book.rb

```diff
@@ -14,15 +19,14 @@ class Book < ApplicationRecord
   validates :isbn13, length: { is: 13 },
                      numericality: { only_integer: true, greater_than: 0 },
                      allow_blank: true
-  validates :title, :author, :status, presence: true
-
-  private
+  validates :title, :first_name, :last_name, :status, presence: true
 
-  def strip_input_fields
-    self.isbn10 = isbn10.strip unless isbn10.nil?
-    self.isbn13 = isbn13.strip unless isbn13.nil?
+  def decorate
```

Comment:
Hmm, the model should not really know about any decorator-related logic. A standard way of adding decorator-related logic in Rails is to implement something similar to the view helpers. I recommend checking out [this](https://github.com/drapergem/draper) gem, it simplifies the process of building and accessing decorated objects. You can of course, also build this functionality from scratch but Draper simplifies this process.